docs/content.zh/docs/concepts/architecture.md

```diff
@@ -24,57 +24,65 @@ specific language governing permissions and limitations
+<a name="control-loop"></a>
 
-Users can interact with the operator using the Kubernetes command-line tool, [kubectl](https://kubernetes.io/docs/tasks/tools/). The Operator continuously tracks cluster events relating to the `FlinkDeployment` and `FlinkSessionJob` custom resources. When the operator receives a new resource update, it will take action to adjust the Kubernetes cluster to the desired state as part of its reconciliation loop. The initial loop consists of the following high-level steps:
+## 控制平面
+Operator 遵循 Kubernetes 原则，特别是 [控制平面](https://kubernetes.io/docs/concepts/architecture/controller/)：
 
-1. User submits a `FlinkDeployment`/`FlinkSessionJob` custom resource(CR) using `kubectl`
-2. Operator observes the current status of the Flink resource (if previously deployed)
-3. Operator validates the submitted resource change
-4. Operator reconciles any required changes and executes upgrades
+{{< img src="/img/concepts/control_loop.svg" alt="控制循环" >}}
 
-The CR can be (re)applied on the cluster any time. The Operator makes continuous adjustments to imitate the desired state until the current state becomes the desired state. All lifecycle management operations are realized using this very simple principle in the Operator.
+用户可以使用 Kubernetes 命令行工具 [kubectl](https://kubernetes.io/docs/tasks/tools/) 与操作员进行交互。Operator 不断跟踪与 `FlinkDeployment` 和 `FlinkSessionJob` 自定义资源相关的集群事件。当操作员接收到新的资源更新时，它将采取行动，以调整 Kubernetes 集群以达到所需状态，作为其协调循环的一部分。初始循环包括以下高级步骤：
 
-The Operator is built with the [Java Operator SDK](https://github.com/java-operator-sdk/java-operator-sdk) and uses the [Native Kubernetes Integration](https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/resource-providers/native_kubernetes/) for launching Flink deployments and submitting jobs under the hood. The Java Operator SDK is a higher level framework and related tooling to support writing Kubernetes Operators in Java. Both the Java Operator SDK and Flink's native kubernetes integration itself is using the [Fabric8 Kubernetes Client](https://github.com/fabric8io/kubernetes-client) to interact with the Kubernetes API Server.
+1. 用户使用 `kubectl` 提交 `FlinkDeployment`/`FlinkSessionJob` 自定义资源（CR）
+2. Operator观察 Flink 资源的当前状态（如果先前已部署）
+3. Operator验证提交的资源更改
+4. Operator协调任何必要的更改并执行升级
 
-## Flink Resource Lifecycle
+CR 可以随时在集群上（重新）应用。Operator 通过不断调整来模拟所需状态，直到当前状态变为所需状态。Operator 中的所有生命周期管理操作都是使用这个非常简单的原则实现的。
 
-The Operator manages the lifecycle of Flink resources. The following chart illustrates the different possible states and transitions:
+Operator [Java Operator SDK](https://github.com/java-operator-sdk/java-operator-sdk)和使用 [Native Kubernetes Integration](https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/resource-providers/native_kubernetes/) 在幕后启动 Flink 部署和提交作业。Java Operator SDK 是一个更高级别的框架和相关工具，用于支持使用 Java 编写 Kubernetes Operator。Java Operator SDK 和 Flink 的本地 Kubernetes 集成本身都使用 [Fabric8 Kubernetes Client](
```

Comment:
```suggestion
Operator 使用 [Java Operator SDK](https://github.com/java-operator-sdk/java-operator-sdk) 构建，并使用 [Native Kubernetes Integration](https://nightlies.apache.org /flink/flink-docs-master/docs/deployment/resource-providers/native_kubernetes/) 用于启动 Flink deployment 并在后台提交作业。
Java Operator SDK 是一个更高级别的框架和相关工具，用于支持使用 Java 编写 Kubernetes Operator。Java Operator SDK 和 Flink 的原生 kubernetes 集成本身都使用 [Fabric8 Kubernetes 客户端](https://github.com/fabric8io/kubernetes-client) 与 Kubernetes API 服务器交互。
```

`The Operator is built with the ` and `to interact with the Kubernetes API Server.` are missed here.

Also, how about leave deployment instead of `部署`?